installer/pkg/components/database/cloudsql/init/00-create-and-init-sessions-db.sql

```diff
@@ -0,0 +1,16 @@
+-- Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+-- Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+-- must be idempotent
+
+CREATE DATABASE IF NOT EXISTS `gitpod-sessions` CHARSET utf8mb4;
```

Comment:
We could - not necessarily in this PR - move this script (these scripts) to something like `pkg/components/database/init`

Reply:
Possible, but the init scripts are subtly different.